thirdparty/README.md

```diff
@@ -480,6 +480,20 @@ Files extracted from upstream source:
 - License.txt
 
 
+## Rvo2-3D
+
+- Upstream: http://gamma.cs.unc.edu/RVO2/
+- version: 1.0.1
+- License: Apache 2.0
+
+Files extracted from upstream source:
+
+- All .cpp and .h files in the `src/` folder except for RVOSimulator.cpp and RVOSimulator.h
```

Comment:
You should also backport #36102, so remove `RVO.h` too and problematic stuff from `API.h`.